[
    {
        "additional_details": null,
        "exception": "Could not fetch VPC configuration: An error occurred (UnauthorizedOperation) when calling the DescribeRegions operation: You are not authorized to perform this operation.",
        "file": "services.py",
        "line": 56,
        "traceback": "Traceback (most recent call last):\n  File \"/Users/sudharsan/audit_poc/ScoutSuite/ScoutSuite/providers/aws/facade/base.py\", line 201, in build_region_list\n    ec2_not_opted_in_regions = self.session.client('ec2', 'us-east-1') \\\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/botocore/client.py\", line 386, in _api_call\n    return self._make_api_call(operation_name, kwargs)\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/botocore/client.py\", line 705, in _make_api_call\n    raise error_class(parsed_response, operation_name)\nbotocore.exceptions.ClientError: An error occurred (UnauthorizedOperation) when calling the DescribeRegions operation: You are not authorized to perform this operation.\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/Users/sudharsan/audit_poc/ScoutSuite/ScoutSuite/providers/base/services.py\", line 56, in _fetch\n    await service_config.fetch_all(**method_args)\n  File \"/Users/sudharsan/audit_poc/ScoutSuite/ScoutSuite/providers/aws/resources/regions.py\", line 14, in fetch_all\n    for region in await self.facade.build_region_list(self.service, regions, excluded_regions, partition_name):\n  File \"/Users/sudharsan/audit_poc/ScoutSuite/ScoutSuite/providers/aws/facade/base.py\", line 229, in build_region_list\n    raise e\n  File \"/Users/sudharsan/audit_poc/ScoutSuite/ScoutSuite/providers/aws/facade/base.py\", line 208, in build_region_list\n    self.session.client('ec2', self.session.region_name). \\\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/botocore/client.py\", line 386, in _api_call\n    return self._make_api_call(operation_name, kwargs)\n  File \"/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/botocore/client.py\", line 705, in _make_api_call\n    raise error_class(parsed_response, operation_name)\nbotocore.exceptions.ClientError: An error occurred (UnauthorizedOperation) when calling the DescribeRegions operation: You are not authorized to perform this operation.\n"
    }
]
